reconcile/kafka_clusters.py

```diff
@@ -75,10 +76,26 @@ def run(dry_run, thread_pool_size=10,
 
     current_state = ocm_map.kafka_cluster_specs()
     desired_state = fetch_desired_state(kafka_clusters)
+    kafka_service_accounts = ocm_map.kafka_service_account_specs()
 
     error = False
     for kafka_cluster in kafka_clusters:
         kafka_cluster_name = kafka_cluster['name']
+        # get a service account for the cluster
+        # we match cluster to service account by name
+        service_accounts = [sa for sa in kafka_service_accounts
+                            if sa['name'] == kafka_cluster_name]
+        if service_accounts:
+            service_account = service_accounts[0]
```

Comment:
Please, please, please, let's avoid variables that differ from just one character. As we get tired we'll confuse them.

In this context, `sa` or `acc` are perfectly valid identifiers.

Reply:
ok!